docs/api/create_docker_container.md

```diff
@@ -0,0 +1,78 @@
+# Creating a container
+
+Testcontainers' generic container support offers the most flexibility, and makes it easy to use virtually any container images as temporary test environments. To interact or exchange data with a container, Testcontainers for .NET provides an implementation of the `TestcontainersBuilder<TDockerContainer>` to configure and create the resource.
+
+## Examples
+
+A NGINX container that binds the HTTP port and hosts static content. Hit the static content in your browser at `http://localhost:80` or `http://host-ip:80`.
+
+```csharp
+await new TestcontainersBuilder<TestcontainersContainer>()
```

Comment:
Can we get all code examples in the docs from actual code that is executed as part of the build, or is it too much for our first iteration on the docs?

Java does it, Go not yet.

Reply:
I don't like to do it now. With the dedicated modules, I will restructure the tests, then we can introduce it.

Reply:
Alright, then let's do it like this for the first iteration 👍